package-scripts/datadog-agent/preinst

```diff
@@ -42,7 +42,15 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
     if $( apt-key finger | grep -q 'A292 3DFF 56ED A6E7 6E55  E492 D3A8 0E30 382E 94DE' ); then
         echo "... key already installed"
     else
-        apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 382E94DE >/dev/null 2>&1 && echo " ... OK" || echo " ... failed"
+        apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 382E94DE >/dev/null 2>&1
+        if [ $? -eq 0 ]; then
+            echo " ... OK"
+        else
+            echo " ... failed"
+            if [ -f "/etc/debian_version" ] || [ "$DISTRIBUTION" = "Debian" ]; then
```

Comment:
since `/etc/debian_version` is also present on Ubuntu, this condition will also be true on Ubuntu.

I think we can either:
1. only use the `$DISTRIBUTION` value, which should work as long as `lsb_release` is actually present on the system (we can enforce that by declaring `lsb-release` as a dependency of the package), or
2. change the error message a bit, to something like: `If you're running Debian 9, [...]`

Honestly I think option `2.` is enough :)